.github/workflows/pull-request-check-rust-api.yaml

```diff
@@ -101,8 +101,15 @@ jobs:
       # We won't be running any of the regular regression tests, as these are covered
       # by the other jobs already present in `pull-request-checks.yaml`.
       - name: Run Rust API tests
+        env:
+          # This is needed because an issue on the CI runner is preventing the link-cplusplus crate
+          # cxx.rs depends on to select the correct default (libc++, the clang++ standard library)
+          # on this platform.
+          # (https://docs.rs/cc/latest/cc/struct.Build.html#method.cpp_link_stdlib)
+          CXXSTDLIB: c++
```

Comment:
Is this change actually needed?

Reply:
Not really - was a speculative fix but didn't work. Will drop.